docs/fleet/locations.md

```diff
@@ -54,18 +54,30 @@ To move a sub-location to the top level of locations, select **Root** from the *
 A location always belongs to the organization it was created in.
 Members of the organization have access to all locations in the organization by default.
 
-You can share a location beyond its organization by [sharing a location with an additional organization](#share-a-location-with-an-additional-organization).
+You can share a location beyond its organization by sharing a location with an additional organization:
 
 #### Share a location with an additional organization
 
 Share your location with another organization you belong to by selecting the organization from the **Add Organization** dropdown menu and clicking **Share**.
 
+To share your location with an organization you are not a member of, enter the organization ID (a string like `1ab2c3d1-1234-123a-abcd-abcdef123456`) and click **Share**.
+Members of the other org can find the org ID on their org settings page.
+
 ![The Add Organization dropdown in the Viam app displays all organizations the user is a member of.](/fleet/app-usage/add-org-drop-down.png)
 
-The location's page lists newly added organization, along with the organization identified as the **primary owner**:
+The location's page lists the newly added organization, along with the organization identified as the _primary owner_:
 
 ![After adding another org, the Viam app lists it under the orgs that share this location list.](/fleet/app-usage/after-add-org.png)
 
+The list of organizations includes the names of organizations to which you belong, and the org IDs of organizations to which you do not belong.
+
+{{% alert title="Info" color="info" %}}
+
+Only members of the _primary owner_ organization can share the location.
+If the location has been shared with additional organizations, members of those secondary organizations cannot share the location.
```

Comment:
This is not correct, they can as long as they are org owners in their org

Reply:
So if Natalia's Org owns Location 123, and you share Location 123 with Jessamy's Org, then I can share Location 123 with Brunhilda's Org? And then can Brunhilda share Location 123 with Damian's Org or does the chain stop between secondary and tertiary org owners?

Reply:
```suggestion
If the location has been shared with additional organizations, owners of those secondary organizations can also share the location.
```